src/components/governance/PageProposal.vue

```diff
@@ -13,6 +13,15 @@
             {{ status.badge }}
           </span>
           <h2 class="proposal-title">{{ proposal.title }}</h2>
+          <p class="proposer">
+            <template v-if="proposal.proposer !== `unknown`">
+              Proposed by
+              <Bech32 :address="proposal.proposer" :long-form="true" />
```

Comment:
why long form?

Reply:
This should also resolve to a validator moniker if it is a validator

Reply:
My fault, I am a complete address fan :-) I am going to remove the long-form prop and figure out how to add the moniker.

Reply:
As @faboweb suggested and to following our new `please_short_pull_requests` rule :-), let's finish this pr and create a new issue for resolve the address to a moniker if the proposer is a validator.

Reply:
`please_short_pull_requests` 💯💯💯💯